rfcs/0074-community-coordination-hub.md

```diff
@@ -0,0 +1,200 @@
+for coordinating the community's efforts.
+
+It shall make it easy to mentally grasp how the community coordinates as a whole.
+Getting involved with the community should feel as natural as skimming through
+code on github (or on a local clone).
+
+It shall be a place where established truths or conclusions that (have) emerge
+from the communities conversations and decision making processes are recorded.
+
+Therefore it shall act as single source of truth for aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall _not_:
+
+- document the standard library (or any other highly specific and established technical subject).
+- act as discussion forum (but document outcomes).
+- implement code (but coordinate it's implementation).
+
+To implement, myself, and anyone interested, will review https://github.com/kubernetes/community
+to the smallest details.
+
+Based on this review, we'll create a clean new repository at
+`NixOS/community`, with what we considered was worth porting from
+[Kubernetes' community portal](https://github.com/kubernetes/community).
+
+As a general guideline for this review, we'd acknowledge the success
+and impressive dynamics of the Kubernetes ecosystem.
+
+## Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas we might put into `NixOS/community` collected during
+this RFC's review process.
+
+- [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+- Explain and Support an Efficient Code Review Process.
+- Move non-technical aspects of nixpkgs' manual here
+- Record Meeting Minutes of working groups
+- Evolve Design Prototypes before moving to RFC with in special interest groups
+
+# User Stories
+[user-stories]: #user-stories
+
+If this RFC is implemented ...
+
+## As a newcomer
+As a newcomer, I'd be able to quickly find my way around, and start beeing
+useful before beeing turned down by inconsistent and spread-out onboarding.
+
+## As a veteran
+As a veteran, I'd be able to promote some ideas, that I care about,
+towards completion while suffering from less friction and overly complex
+coordination. I can leverage my experience and skills more effectively
+for the benefit of the community.
+
+## As a mentor
+As a mentor, I can inform my efforts with a clear and consistent reference.
+It helps me to more effectively maintaining those high quality human 
+interactions that the NixOS community is known for.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+There _can_ be a better solution to achieve _optimum levels of empowerment_
+for our community. We _could just **wait**_ for them to unveil themselves.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- [Benevolent Dictatorship](https://discourse.nixos.org/t/what-would-you-do-if-you-were-the-bdfl-of-nix/6949)
+- Ephemeral Conversations (Discourse)
```

Comment:
See my other comment. I think this RFC conflates decision making processes and the presentation of the results of a decision making process (unless I understood the goal of the hub incorrectly). If the goal is the latter, an alternative would be:

```
- Extend the nixpkgs manual to contain missing information for contributors.
```

Reply:
Addressed in: 
- https://github.com/NixOS/rfcs/commit/72b3e13079a140553e4dc92a0bf783ad679695f5
- https://github.com/NixOS/rfcs/commit/ca1a6a2b6fd45af74b2755fdefe96fff2dd2167e
- https://github.com/NixOS/rfcs/commit/064852dd02e336c08fd5823daf323a055ce7f88a